js/aax.js

```diff
@@ -2376,7 +2376,7 @@ module.exports = class aax extends Exchange {
         //     "ts": 1573561743499
         // }
         const data = this.safeValue (response, 'data', []);
-        return this.parseTransactions (data, code, since, limit);
+        return this.parseTransactions (data, currency, since, limit);
```

Comment:
> a bug (`code` instead of `currency`) was spread across different exchange implementations.

That isn't a bug, since parseTransactions expects a currency structure in the second argument, not a currency code.

Reply:
Actually, nevermind, i misread the commit, indeed, you're right, it's a bug.